lib/runtime.js

```diff
@@ -100,18 +98,17 @@ exports.enable = function (Module) {
   // export statement. The keys of the getters object are exported names,
   // and the values are functions that return local values.
   Mp.export = function (getters) {
-    var module = this;
-    setESModule(module);
+    setESModule(this);
 
-    if (utils.isPlainObject(getters)) {
-      Entry.getOrCreate(module.id).addGetters(getters);
+    if (getters && typeof getters === "object") {
+      Entry.getOrCreate(this.id).addGetters(getters);
```

Comment:
Instead of the heavier `utils.isPlainObject` I went with the lighter object check used everywhere else in the code.